src/compute-pr-actions.ts

```diff
@@ -221,7 +221,7 @@ function createWelcomeComment(info: PrInfo) {
         introCommentLines.push(" " + dangerComment);
     }
 
-    const waitingOnThePRAuthorToMerge = !info.hasMergeConflict && info.travisResult === TravisResult.Pass && info.dangerLevel === "ScopedAndTested" && hasFinalApproval(info)
+    const waitingOnThePRAuthorToMerge = !info.hasMergeConflict && info.travisResult === TravisResult.Pass && info.dangerLevel === "ScopedAndTested" && hasFinalApproval(info).approved

```

Comment:
The actual change - I switched to `hasFinalApproval` to return an object instead of a boolean but never reached in to get back out down here